micromixin-transformer/src/main/java/org/stianloader/micromixin/transform/api/SlicedInjectionPointSelector.java

```diff
@@ -64,33 +64,6 @@ public SlicedInjectionPointSelector(@NotNull InjectionPointSelector selector, @N
         this.unsafe = unsafe;
     }
 
-    /**
-     * Obtains the instruction immediately after the next instruction.
-     *
-     * <p>Handle with care, improper use may lead to unexpected behaviour with frames or jumps!
-     * The method is mainly intended to be used in order to easily check whether an instruction lies between
-     * two other instructions.
-     *
-     * @param method The method to find the entrypoints in.
-     * @param remapper The remapper instance to make use of. This is used to remap any references of the mixin class to the target class when applying injection point constraints.
-     * @param sharedBuilder Shared {@link StringBuilder} instance to reduce {@link StringBuilder} allocations.
-     * @return Instruction immediately after the next instruction.
-     */
-    @Nullable
-    public AbstractInsnNode getAfterSelected(@NotNull MethodNode method, @NotNull SimpleRemapper remapper, @NotNull StringBuilder sharedBuilder) {
```

Comment:
Removing this method isn't that great as that is technically part of public API, so would necessitate bumping the version to 0.7.0